st2common/st2common/models/db/action.py

```diff
@@ -215,6 +209,9 @@ class ActionAliasDB(stormbase.StormBaseDB):
     formats = me.ListField(
         field=me.StringField(),
         help_text='Possible parameter formats that an alias supports.')
+    file_uri = me.StringField(
```

Comment:
Hm, this field with the same constructor arguments is already declared on `stormbase.StormContentDB` so it shouldn't be needed here, right?